extensions/sql-database-projects/README.md

```diff
@@ -22,6 +31,19 @@ Please report issues and feature requests [here.](https://github.com/microsoft/a
 * Start from an existing database by using the `Create Project from Database` from the command palette or database context menu.
 * Start from an OpenAPI/Swagger spec by using the `Generate SQL Project from OpenAPI/Swagger spec` command (Preview).
 
+## Settings
+
+### General Settings
+- `sqlDatabaseProjects.dotnetSDK Location`: The path to the dotnet SDK. If not set, the extension will attempt to find the dotnet SDK on the system.
```

Comment:
Can you clarify that this is the path to the _folder containing_ `dotnet`/`dotnet.exe`, not the path to the binary itself?